bin/lov_migrator

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env ruby
+
+# Exit cleanly from an early interrupt
+Signal.trap("INT") { exit 1 }
+
+require 'optparse'
+
+options = {}
+OptionParser.new do |opts|
+  opts.banner = "Usage: lov_migrator [options]"
+  opts.on( '-a', '--all') do 
+    options[:vocabs] = [:all]
+  end
+  opts.on('-v', '--vocabularies PREFIX1, PREFIX2', 'Comma-separated list of vocabularies to update or import') do |acronym|
+    ontologies_acronyms = acronym
+  end
+  opts.on('--dispatch-csv FILENAME', 'Specify the CSV file to dispatch') do |csv_file|
+    options[:csv_dispatch_filename] = csv_file
+  end
+  # Display the help screen, all programs are assumed to have this option.
+  opts.on( '-h', '--help', 'Display this screen' ) do
+    puts opts
+    exit
+  end
+end.parse!
+
+raise OptionParser::MissingArgument if options[:vocabs].nil?
+
+require 'open-uri'
+require 'net/http'
+require 'json'
+require 'date'
+require 'benchmark'
+require 'csv'
+LOV_ENDPOINT = "https://lov.linkeddata.es/dataset/lov"
+CSV_ADDED_ATTRS = [ 'destination', 'who', 'comment' ]
+CSV_DISPATCH_FILENAME = 'LOV_vocabularies.csv'
+CSV_FILENAME = "vocabs.csv"
+
+module LOVMigrator
+
+  attr_accessor :lov_endpoint
+  class SparqlParser
+    attr_accessor :last_processed_date
+    def initialize(endpoint = LOV_ENDPOINT)
+      @lov_endpoint = endpoint
+      @last_processed_date_file = ".last_processed_date.txt"
```

Comment:
- We can't be sure that the file won't disappear, i'll keep it temporarily until we have a running instance of LovPortal, I'll bring the last processed date from there.
- I'll use a full path so that the file will be stored in the same place

Reply:
if the file disappear the script will still work and update the csv

Reply:
why not pass it as an argument of the script, the user will know, which date will start the import.

Reply:
TODO